src/screens/LunchScreen.js

```diff
@@ -124,11 +122,11 @@ export const LunchScreen = ({ navigation }) => {
       <FlatList
         refreshControl={
           <RefreshControl
-            colors={[colors.accent]}
-            onRefresh={() => refetch?.()}
             // using this refreshing prop causes the loading spinner to animate in from top and
             // push the list down, whenever we change the date
             refreshing={false}
+            onRefresh={() => refetch?.()}
+            colors={[colors.accent]}
             tintColor={colors.accent}
```

Comment:
what is the reason for this resorting of props?

Reply:
see commit message:

> ordered props for `RefreshControl` like everywhere else for `RefreshControl`